build/gulp/plugins/gulp-react-docgen.ts

```diff
@@ -19,11 +19,18 @@ export default () =>
       return
     }
 
-    try {
-      const contents = getComponentInfo(file.path)
+    const infoFilename = file.basename.replace(/tsx$/, 'info.json')
```

Comment:
would suggest to make expression more explicit by using dot at start:

```js
 const infoFilename = file.basename.replace(/\.tsx$/, '.info.json')
```

Reply:
Agree, will be done tomorrow